### PR TITLE
Modifications to fix downstream problems in parsons

### PR DESCRIPTION
**Pull request by andrie**

### CODE REVIEW

File: R/bucket_list.R [hunk 1]
```diff
@@ -61,7 +74,9 @@ bucket_list <- function(
   group_name,
   group_put_max = rep(Inf, length(labels)),
   options = sortable_options(),
-  class = "default-sortable"
+  class = "default-sortable",
+  orientation = c("horizontal", "vertical"),
+  dependencies = NULL
```
Comment (schloerke) [inline R/bucket_list.R]: let's remove this parameter.

File: inst/htmlwidgets/plugins/sortable-rstudio/bucket_list.scss [hunk 2]
```diff
@@ -9,8 +9,21 @@
...
+    // flex-flow: column nowrap;
+  }
+
+  .bucket-list.horizontal {
     flex-direction: row;
-    flex-wrap: wrap;
+  }
+
+  .bucket-list.vertical {
+    flex-direction: column;
```
Comment (schloerke) [inline inst/htmlwidgets/plugins/sortable-rstudio/bucket_list.scss]: add `flex-wrap: nowrap`?

File: inst/htmlwidgets/plugins/sortable-rstudio/rank_list.css [hunk 3]
```diff
@@ -27,51 +26,53 @@
   background-color: #f5f5f5;
   border: 1px solid #ddd;
   overflow: hidden;
-  width: 100%;
+  width: fit-content;
```
Comment (schloerke) [inline inst/htmlwidgets/plugins/sortable-rstudio/rank_list.css]: let's leave it at 100% for now.

File: inst/htmlwidgets/plugins/sortable-rstudio/rank_list.scss [hunk 4]
```diff
@@ -8,7 +8,7 @@
     border: 1px solid $border-color;
     padding: 10px;
     margin: 5px;
-    display: flex;
+    // display: flex;
```
Comment (schloerke) [inline inst/htmlwidgets/plugins/sortable-rstudio/rank_list.scss]: if we keep this one, side by side boxes will wrap to the next row on a skinny window, I believe

File: R/bucket_list.R (hunk 1 above)
Comment (andrie) [reply R/bucket_list.R]: Agreed. I'm removing the `dependencies` argument and will create instructions for downstream users to add their own CSS.

File: inst/htmlwidgets/plugins/sortable-rstudio/bucket_list.scss (hunk 2 above)
Comment (andrie) [reply inst/htmlwidgets/plugins/sortable-rstudio/bucket_list.scss]: Agreed.

File: inst/htmlwidgets/plugins/sortable-rstudio/rank_list.scss (hunk 4 above)
Comment (andrie) [reply inst/htmlwidgets/plugins/sortable-rstudio/rank_list.scss]: Thanks.

**Comment by andrie** [comment]
I have addressed the comments in cd4f5a6ff45e700853bd5be4bcc1d2f820819f0f

File: R/dependencies.R [hunk 5]
```diff
@@ -1,24 +1,22 @@
-css_dependency <- function(name, files) {
-  htmltools::htmlDependency(
-    name,
-    utils::packageVersion("sortable"),
-    "htmlwidgets/plugins/sortable-rstudio",
-    package = "sortable",
-    stylesheet = files
+css_dependency <- function(name, files, package = "sortable") {
```
Comment (schloerke) [inline R/dependencies.R]: is the package arg needed here?

File: inst/htmlwidgets/plugins/sortable-rstudio/rank_list.css [hunk 6]
```diff
@@ -4,7 +4,6 @@
   border: 1px solid #ddd;
   padding: 10px;
   margin: 5px;
-  display: flex;
```
Comment (schloerke) [inline inst/htmlwidgets/plugins/sortable-rstudio/rank_list.css]: this one is needed to have the horizontal containers wrap to the bottom on a skinny window, correct?

**Pull request merged**